[2024-05-29T19:41:54.342+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 1_dag.is_api_available manual__2024-05-29T19:41:11.590015+00:00 [queued]>
[2024-05-29T19:41:54.394+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: 1_dag.is_api_available manual__2024-05-29T19:41:11.590015+00:00 [queued]>
[2024-05-29T19:41:54.395+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-05-29T19:41:54.396+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-05-29T19:41:54.398+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-05-29T19:41:54.591+0000] {taskinstance.py:1383} INFO - Executing <Task(HttpSensor): is_api_available> on 2024-05-29 19:41:11.590015+00:00
[2024-05-29T19:41:54.717+0000] {standard_task_runner.py:55} INFO - Started process 6020 to run task
[2024-05-29T19:41:54.771+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', '1_dag', 'is_api_available', 'manual__2024-05-29T19:41:11.590015+00:00', '--job-id', '26', '--raw', '--subdir', 'DAGS_FOLDER/1_dag.py', '--cfg-path', '/tmp/tmpmwqxmqps']
[2024-05-29T19:41:54.785+0000] {standard_task_runner.py:83} INFO - Job 26: Subtask is_api_available
[2024-05-29T19:41:55.560+0000] {task_command.py:376} INFO - Running <TaskInstance: 1_dag.is_api_available manual__2024-05-29T19:41:11.590015+00:00 [running]> on host 4b56679d13b8
[2024-05-29T19:41:55.798+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=1_dag
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2024-05-29T19:41:11.590015+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-29T19:41:11.590015+00:00
[2024-05-29T19:41:55.803+0000] {http.py:120} INFO - Poking: api/
[2024-05-29T19:41:55.953+0000] {base.py:71} INFO - Using connection ID 'user_api' for task execution.
[2024-05-29T19:41:56.118+0000] {http.py:148} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2024-05-29T19:41:57.875+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2024-05-29T19:41:58.089+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=1_dag, task_id=is_api_available, execution_date=20240529T194111, start_date=20240529T194154, end_date=20240529T194158
[2024-05-29T19:41:58.210+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-05-29T19:41:58.381+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
